.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,37 @@
+<!--
+Title: Conventional Commits format, type from the Type Table in
+.standards/docs/standards/github.md (e.g. feat(auth): implement password recovery).
+Labels: add one type label (feat, fix, ...) and one complexity label (patch, minor, major).
+Replace every HTML comment below with real content before requesting review.
+-->
+
+## 1. Context
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)